forage-android/src/main/java/com/joinforage/forage/android/ui/ForageElement.kt

```diff
@@ -15,6 +24,8 @@ import com.joinforage.forage.android.core.element.state.ElementState
 interface ForageElement {
     var typeface: Typeface?
 
+    fun setForageContext(forageContext: ForageContext)
```

Comment:
```suggestion
    fun setForageConfig(forageConfig: ForageConfig)
```

Reply:
Changed everything from ForageContext -> ForageConfig in [265eb0a](https://github.com/teamforage/forage-android-sdk/pull/97/commits/265eb0aced8e8269f916f88e2d4cff9390b0bfe7). Marking as resolved